src/util/pkce.ts

```diff
@@ -1,18 +1,19 @@
 /**
- * Helper class for generating verifier and challenge strings used for
+ * Helper class for generating verifier, challenge and state strings used for
  * Proof Key for Code Exchange (PKCE).
  */
 abstract class PKCE {
+  /**
+   * Creates a random state.
+   */
+  public static createState(): string {
+    return this.createRandomString(16);
+  }
   /**
    * Creates a random verifier.
-   *
-   * @param size The length of the verifier (default 32 characters)
    */
-  public static createVerifier(size = 32): string {
-    const array = new Uint8Array(size);
-    window.crypto.getRandomValues(array);
-    const verifier = this.base64UrlEncode(array);
-    return verifier;
+  public static createVerifier(): string {
```

Comment:
It won't break anything, but if there's an off chance that someone is using this directly from the SDK for some other purpose, I guess it would not return what they asked for ... okay, I'll put it back. I just didn't like how this was configurable in the first place.